app/src/main/res/values/strings.xml

```diff
@@ -517,4 +516,82 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="questList_disabled_in_country">Never shown in %s</string>
     <string name="quest_toiletAvailability_rest_area_title">Does this rest area have a toilet?</string>
     <string name="quest_maxspeed_name_title">What is the speed limit sign for %s?</string>
+
+    <string name="quest_buildingType_title">"What kind of building is this?"</string>
+    <string name="quest_building_type_generic_building_confirmation">Are you sure that you can not determine the building type more specifically?</string>
+    <string name="quest_building_type_invalid_value">Please select a more specific value! The value you selected is not valid.</string>
+    <string name="quest_buildingType_answer_input_description">Can not determine more specifically</string>
+    <string name="quest_buildingType_comment_title">Describe the building</string>
+    <string name="quest_buildingType_comment_description">Be concise and write a short description of what the intention of the building is</string>
+    <string name="quest_buildingType_comment_emptyAnswer">You need to describe what you can see. If you have no idea what the building is used for, you can make a guess</string>
```

Comment:
emptyAnswer should end with a period (like its first sentence). description just above should also end with a period. (Minor, I know.)